docs/DeveloperInformation.md

```diff
@@ -81,12 +81,11 @@ This project uses [Rake](http://rake.rubyforge.org/) to run tasks from the termi
  
 ### Modify the code
 
-As you add to/modify the code, please try to fit changes into the current structure rather than bolting things on willy-nilly.  See the {file:docs/RepositoryStructure.md Repository Structure page} to see how the code is organized.  If you don't understand something or want to discuss your plan before you get started, contact <mailto:andrew.parker@nrel.gov>.
+As you add to/modify the code, please try to fit changes into the current structure rather than bolting things on willy-nilly.  See the {file:docs/RepositoryStructure.md Repository Structure page} to see how the code is organized.  If you don't understand something or want to discuss your plan before you get started, contact <mailto:matthew.dahlhausen@nrel.gov>.
 
 1. Make a new branch for your changes.
 2. Modify the code on your branch.
-3. Modify the [OpenStudio_Standards Google Spreadsheet](https://drive.google.com/drive/folders/1x7yEU4jnKw-gskLBih8IopStwl0KAMEi?usp=sharing)
- - To get edit access to this spreadsheet, contact <mailto:Marlena.Praprost@nrel.gov>.
+3. If you have data, modify the .json files and run commands to update the database, as appropriate. Historically, openstudio-standards data used a series of google spreadsheets, and is still used for non-90.1- standards. See [OpenStudio_Standards Google Spreadsheet](https://drive.google.com/drive/folders/1x7yEU4jnKw-gskLBih8IopStwl0KAMEi?usp=sharing). Contact <mailto:matthew.dahlhausen@nrel.gov> for edit access.
 4. `bundle exec rake data:update` to download the latest version of the spreadsheet from Google Drive and export the JSON files.
```

Comment:
If you provide edit access, developers will still need to obtain a 'client_secret.json' credential file in order for the rake task to work. Maybe this section should include the fact that you can download the spreadsheets and use `bundle exec rake data:update:manual` to update the jsons from the local copies.

Reply:
Added further detail with [1a3e14a](https://github.com/NREL/openstudio-standards/pull/1576/commits/1a3e14a50c198932756de68a7cc658bbdef47614).